base/server/cms/src/com/netscape/cms/servlet/admin/AdminServlet.java

```diff
@@ -992,7 +994,7 @@ protected String auditParams(HttpServletRequest req) {
             if (name.equals(Constants.RS_ID)) continue;
 
             String value = null;
-            if (name.equalsIgnoreCase("PASSWORD_CACHE_ADD"))
+            if (StringUtils.containsIgnoreCase(name, "pass"))
```

Comment:
So the one concern I have is that this isn't consistent with the other usage here:

https://github.com/dogtagpki/pki/blob/e90fb56b4db39eccab176dd358f35af59160ad3f/base/server/cms/src/com/netscape/cms/servlet/admin/AdminServlet.java#L203-L225

I'd personally like to see this check refactored into a function (`isSensitive` perhaps?) and then utilized in both places to keep the logic the same.